### DISCUSSION TITLE
[Bug]: the code can not building,missding same codes in version 0.36.1

### DISCUSSION THREAD

**Discussion by classcit** [strimzi/strimzi-kafka-operator#9019] [General]
### Bug Description

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/90080178/4da45301-f560-41b9-b17d-fdcce829453b)

like this

### Strimzi version

0.36.1

### Kubernetes version

1.21

**Comment by scholzj** [comment]
These classes are auto-generated as part of the build. So perhaps your IDE did not load them properly or you did not actually build it?
